package.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
javascript lol. It's conventional to commit this and the accompanying 3K lines of json lockfile, but it feels out of place in a golang project, where the npm dependencies are used only for the test environment and not in build output. Not sure if there's a better way to structure these test dependencies.